tools/git-hooks/Readme.md

```diff
@@ -0,0 +1,33 @@
+# Git Hooks for iceoryx
+
+When developing on iceoryx, git-hooks are doing some sanity checks and can execute linters/formatter when you commit code to iceoryx.
+In the current state the git hooks do the following job:
+
+- checking if commit messages follows the format `[iox-#<Github issue number>]` and auto-format them if necessary
+- check if commit message subject and body follow six of the [seven rules of a great commit message](https://chris.beams.io/posts/git-commit/)
```

Comment:
You can omit rule 7 because it is not checked in the commit hooks. Of course you can omit also other rules by just ignoring the warning messages, the hooks will not prevent you from committing.